cloudify_rest_client/manager.py

```diff
@@ -76,6 +76,14 @@ def __init__(self, manager):
         super(ManagerItem, self).__init__()
         self.update(manager)
 
+    @property
+    def id(self):
+        """
+        Manager's ID in the DB (unique, incremental)
+        type: db.Integer
```

Comment:
let's change the type in the docstrings for this whole class to not use db.Text/db.Integer, those types are restservice-specific and are meaningless for the client (use just int/str instead)

Reply:
Fixed